src/sage/rings/function_field/function_field_polymod.py

```diff
@@ -571,7 +571,22 @@ def _repr_(self):
             sage: L._repr_()
             'Function field in y defined by y^5 - 2*x*y + (-x^4 - 1)/x'
         """
-        return "Function field in %s defined by %s" % (self.variable_name(), self._polynomial)
+        return (f"Function field in {self.variable_name()} defined by "
+                f"{self._polynomial}")
+
```

Comment:
make this just one line (one f-string).

Reply:
That would exceed our 80 character line length limit and not comply with the style guide. I'm not personally a fan of line length limits, but I want to make sure you're okay with violating the style guide in this instance before I do that.

Reply:
I was aware of that. The 80 character limit is for readability. In the present case, I think the readability will get better by joining the dangling part into the previous line. Readability is more important than the rule.

My suggestion is a mild one. Taking it or not is just up to you. Let me know which you prefer.

Reply:
Fair. I've taken the suggestion.

Reply:
Thanks.